mergify_engine/tests/unit/test_worker.py

```diff
@@ -1207,10 +1212,10 @@ async def test_stream_processor_retrying_after_read_error(
 
     owners_cache = worker.OwnerLoginsCache()
     p = worker.StreamProcessor(
-        redis_stream, redis_cache, "shared-0", None, owners_cache
+        redis_stream, redis_cache, mock.Mock(), "shared-0", None, owners_cache
     )
 
-    installation = context.Installation(FAKE_INSTALLATION, {}, None, None)
+    installation = context.Installation(FAKE_INSTALLATION, {}, None, None, None)
```

Comment:
Unrelated to your change, but this should not be None in theory. Mypy didn't catch it because the method signature is not typed :(